src/odemis/driver/pigcs.py

```diff
@@ -1606,8 +1607,8 @@ def selfTest(self):
             if self._model in (MODEL_C867,): # support temperature reading
                 # No support for direct open-loop mode
                 # TODO put the temperature as a RO VA?
-                current_temp = float(self.GetParameter(1, 0x57))
-                max_temp = float(self.GetParameter(1, 0x58))
+                current_temp = float(self.GetParameter(str(1), 0x57))
```

Comment:
Yes, I've just checked, and for this controller the "axis" should always be "1". BTW, instead of writing `str(1)`, you can just write `"1"` (or `b"1"` if you want to be a little bit more precise/pedantic).